YapDatabase/YapDatabaseTransaction.m

```diff
@@ -1569,24 +1570,29 @@ - (void)enumerateCollectionsUsingBlock:(void (^)(NSString *collection, BOOL *sto
 	if (statement == NULL) return;
 	
 	YapMutationStackItem_Bool *mutation = [connection->mutationStack push]; // mutation during enumeration protection
-	BOOL stop = NO;
+	__block BOOL stop = NO;
 	
 	// SELECT DISTINCT "collection" FROM "database2";
 	
 	int const column_idx_collection = SQLITE_COLUMN_START;
 	
-	int status;
-	while ((status = sqlite3_step(statement)) == SQLITE_ROW)
-	{
+    __block int status;
```

Comment:
YDB uses tabs, we use spaces, so XCode is jacking up the whitespace.  I'm not inclined to fix it.

Reply:
monster!